FWCore/ParameterSet/src/validateTopLevelParameterSets.cc

```diff
@@ -131,4 +145,13 @@ namespace edm {
     }
   }
 
+  void dumpOptionsToLogFile(unsigned int nThreads,
+                            unsigned int nStreams,
+                            unsigned int nConcurrentLumis,
+                            unsigned int nConcurrentRuns) {
+    LogAbsolute("Options") << "Number of Threads = " << nThreads;
+    LogAbsolute("Options") << "Number of Streams = " << nStreams;
+    LogAbsolute("Options") << "Number of Concurrent Lumis = " << nConcurrentLumis;
+    LogAbsolute("Options") << "Number of Concurrent Runs = " << nConcurrentRuns;
```

Comment:
```suggestion
    LogAbsolute("Options") << "Number of Threads = " << nThreads 
                           << "\nNumber of Streams = " << nStreams
                           << "\nNumber of Concurrent Lumis = " << nConcurrentLumis
                           << "\nNumber of Concurrent Runs = " << nConcurrentRuns;
```
This is much less work for the logging system.

Reply:
Done. Thanks.